cmake/modules/gluecodium/gluecodium/Generate.cmake

```diff
@@ -185,11 +185,14 @@ function(apigen_generate)
   set (_lime_interface_sources "$<FILTER:$<TARGET_PROPERTY:${APIGEN_TARGET},INTERFACE_SOURCES>,INCLUDE,.*\\.lime$>")
   set (_lime_sources "$<FILTER:$<TARGET_PROPERTY:${APIGEN_TARGET},SOURCES>,INCLUDE,.*\\.lime$>")
 
+  set (_gluecodium_args_file "${CMAKE_CURRENT_BINARY_DIR}/gluecodium-command-line.txt")
```

Comment:
This path cannot conflict with other Genium runs?

Reply:
Makes sense in case when one folder contains few calls to apigen_generate.
I'll think about some magic there to make name more unique